packages/utils/src/types/ticket.ts

```diff
@@ -151,7 +151,9 @@ export class Ticket {
   }
 
   recoverSigner() {
-    return new PublicKey(ecdsaRecover(this.signature.signature, this.signature.recovery, this.getHash().serialize()))
+    return PublicKey.deserialize(
```

Comment:
Why not calling `PublicKey.fromSignature` ?

Reply:
good catch, changed.